widgetssdk/src/main/java/com/glia/widgets/internal/authentication/AuthenticationManager.kt

```diff
@@ -19,29 +24,40 @@ internal class AuthenticationManager(
     private val onAuthenticationRequestedCallback: () -> Unit
 ) : Authentication {
     override fun setBehavior(behavior: Authentication.Behavior) {
-        authentication.setBehavior(behavior.toCoreType())
+        try {
+            authentication.setBehavior(behavior.toCoreType())
+        } catch (e: GliaException) {
+            Logger.e(TAG, "Failed to set authentication behavior", e)
```

Comment:
Shall I add this new log to the [Logging events in Mobile SDKs](https://docs.google.com/spreadsheets/d/1ty9bhrbX3VTXBThI5f4GFzy5X-RbEHOCPWfXd7nGzbo/edit?usp=sharing)?